lib/debugger-wrapper.js

```diff
@@ -69,7 +69,7 @@ function instrumentScript(scriptUrl) {
 function setBreakpointOnFunction(scriptUrl, moduleInfo, functionName, functionLocation) {
   const breakpointParameters = {
     lineNumber: functionLocation.start.line - 1, // lines are 0-based in the inspector but 1-based in acorn
-    columnNumber: functionLocation.start.column, // columns are 0-based in both places
+    columnNumber: functionLocation.start.column, // columns are 0-based in both places (?)
```

Comment:
What does `(?)` mean?

Reply:
it means that following my debugging sessions yesterday I'm suddenly not 100% sure `acorn` is 1-based or 0-based